.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/scipy/scipy-mathjax.git
 [submodule "scipy/_lib/boost"]
 	path = scipy/_lib/boost
-	url = https://github.com/scipy/boost-headers-only
+	url = https://github.com/mckib2/boost-headers-only
```

Comment:
I think this is a leftover from local tests?

Reply:
This will need to be changed -- doing some testing to see if we can jump straight to Boost 1.80.0